lib/hammer_cli_foreman/api/session_authenticator_wrapper.rb

```diff
@@ -40,15 +38,13 @@ def user_changed?
       end
 
       def authenticate(request, args)
-        load_session
-
         user = @authenticator.user
 
-        @user_changed ||= (!user.nil? && user != @user)
+        @user_changed ||= (!user.nil? && user != session.user_name)
 
-        if !@user_changed && @permissions_ok && @session_id
+        if !@user_changed && @permissions_ok && session.id
```

Comment:
@rahulbajaj0509 here we should decide if @permissions_ok should be replaced with something like `Sessions.configured?` and block using sessions or if we are okay with the warning. One way or another @permissions_ok was removed and is always unsed resulting in not using sessions at all.